web/styles/informational_overlays.css

```diff
@@ -1,108 +1,186 @@
 .informational-overlays {
+    display: grid;
+    place-items: center;
+
     .overlay-content {
-        /* because zoom breaks at 525px perhaps due to rounding errors, so add a
-           trivial amount of width so it doesn't break. */
-        width: 550px;
+        width: 90%;
+        max-width: 650px;
         margin: 0 auto;
         position: relative;
-        top: calc((30vh - 50px) / 2);
-        border-radius: 4px;
+        top: unset;
+        border-radius: 6px;
         overflow: hidden;
 
         background-color: var(--color-background-modal);
     }
 
     .overlay-tabs {
-        padding: 10px 0;
-        border-bottom: 1px solid hsl(0deg 0% 0% / 20%);
+        position: relative;
+        display: flex;
+        flex-flow: row nowrap;
+        align-items: center;
+        gap: 15px;
+        padding: 0 5%;
+        height: 60px;
+        box-shadow: 0 3px 12px 1px hsl(207deg 100% 11% / 8%);
+        z-index: 2;
 
         .tab-switcher {
-            margin-left: 15px;
+            flex-grow: 1;
         }
 
         .exit {
-            float: right;
-            font-size: 1.5rem;
-            color: hsl(0deg 0% 67%);
-            font-weight: 600;
-            margin: 0 15px;
+            display: inline-block;
+            font-size: 16px;
+            line-height: 1em;
+            text-align: center;
+            width: 1em;
+            color: hsl(0deg 0% 60%);
             cursor: pointer;
+            order: 1;
         }
     }
 
-    .overlay-modal {
-        padding-bottom: 10px;
+    .modal-body {
+        height: 75vh;
```

Comment:
Here again, the only property you really need to set is the `height`! Elements don't have a `max-height` unless you set one elsewhere, so there's no need to unset it. Center-aligning the text has no effect, yet there's a whole bunch of child elements (not of your PR's creation) that have `text-align: left` defined. So you could imagine striking the `text-align` property here, and probably being able to remove the default `text-align: left` from all those child elements (in another PR :)

Reply:
Everything you mentioned in this comment was useless as you suggested but `max-height: unset` is needed as it was getting affected by the `max-height` property of `.modal-body` in `modal.css` and `bootstrap.css` I have also commented the same in code.

Reply:
Awesome! You'll save some future contributor a lot of headaches with comments like that. Thank you :)

Reply:
Yeah, definitely a follow-up item.